### ISSUE TITLE
[New feature] Closure stringify

### ISSUE THREAD

**Issue by leoqbc** [Respect/Stringifier#4]
Hey I hope you guys are doing great! 

I don't know if this will be a good use for you, since this component is part of Respect.

I was looking for a closure stringify, and I notice that we don't have it, and it's not easy to find a good solution for it, for the moment we have something like this:

```php
use function Respect\Stringifier\stringify;

$extern = 2;

$closure = function ($parm1, $parm2) use ($extern) { 
    return $parm1 + $parm2 + $extern;
};


echo stringify($closure); 

/* 
results in: 
`[object] (Closure: { })`
*/
```

What I propose(or something similar):

```php
use function Respect\Stringifier\stringify;

$extern = 2;

$closure = function ($parm1, $parm2) use ($extern) { 
    return $parm1 + $parm2 + $extern;
};


echo stringify($closure); 

/* 
results in: 
`[closure] (function ($parm1, $parm2) use ($extern) { 
    return $parm1 + $parm2 + $extern;
})`
*/
```
I can help on implement it if you've got interested!

What you think about it? @henriquemoody 

All the best!

**Comment by leoqbc** [comment]
In case I've got no response, I'm guessing that there's no interest in it, so I'm closing it.

Best

**Comment by henriquemoody** [comment]
Hi, @leoqbc! 

That was a false assumption. It's just that sometimes it takes some time for me to reply, that's all. 

I find that idea quite nice, because indeed `[object] (Closure: { })` doesn't say that much. I wonder, though, how would we implement that, and if that wouldn't be too expensive as well.

Also, I would like to keep messages in a single line, so perhaps `[closure] (function ($parm1, $parm2) use ($extern) { .. }` would be enough.

Feel free to close this issue if you feel like.

**Comment by henriquemoody** [comment]
Fixed by bc0071e073d5541b76006fcdbe6561bee158966f
